### ISSUE TITLE
# noqa lines are not ignored

### ISSUE THREAD

**Issue by philtay** [public/flake8-import-order#29]
the correctness of an import is checked even if the line is commented with # noqa. it should be skipped.

**Comment by untitaker** [comment]
Can't fully reproduce. The error is raised for "import os", so this yields errors:

    import sys  # noqa
    import os

while this does not:

    import sys
    import os  # noqa

**Comment by philtay** [comment]
Try with 3 to better see the problem:

```python
import argparse
import calendar  # noqa
import base64
```

```
./foo.py:1:1: F401 'argparse' imported but unused
./foo.py:3:1: F401 'base64' imported but unused
./foo.py:3:1: I100 Imports statements are in the wrong order. import base64 should be before import calendar
```

The problem is clearly the second import (calendar). Flake8 does the right thing and ignores the "imported but unused" module, but your plugin still says that they are in wrong order. You should ignore the import where the # noqa comment is added. In the case with only 2 imports it should be irrelevant where the # noqa comment is placed. An error should never be raised.

**Comment by untitaker** [comment]
FWIW I am not a maintainer of this project.

**Comment by philtay** [comment]
To see the problem from another perspective, here I have to use two comments instead of just one where the problem really is:

```python
import calendar
import argparse  # noqa
import base64  # noqa
```

P.S. Got it, you're not a maintainer, it's not your fault, you just work here. Eheheh, I'm joking!

**Comment by public** [comment]
I think I probably assumed that flake8 would take care of that for me :( I think I can implement this though.
